pep/001-initial-pep-process.md

```diff
@@ -0,0 +1,386 @@
+
+1. Reviewer(s) evaluate the implementation. In addition to determining if the
+   implementation is acceptable, they should to identify gaps in the PEP, for
+   example the scope of change was much larger than originally thought, the
+   proposed implementation wasn't actually possible as specified or has
+   problems, etc.
+   
+   The pull request may be closed so the author(s) can go back and iterate on
+   the PEP. We should be careful about follow-up pull requests at this stage so
+   that a PEP isn't incompletely implemented. Critical changes should be made
+   together so limit follow-ups to unrelated changes that not relevant to the PEP.
+
+1. After the implementation is merged, the owner submits a pull request to
+   change the status to `implemented`.
+
+1. Reviewer(s) confirm that the PEP has been completely implemented and no
+   additional work remains, and then merge the pull request. The final state is
+   now **IMPLEMENTED**.
+
+
+#### End States
+
+These are states that terminate the PEP process:
+
+* A PEP that successfully completes the proposal process is **IMPLEMENTED**.
+
+* The PEP may be rejected at certain points by the reviewer. Reviewer should
+  explain the rejection in the active pull request and ultimately close the pull
+  request unmerged. The reviewer submits a pull request updating the status to
+  `rejected` and reiterate the rejection, including enough context that someone
+  can understand it without going back to old pull requests. The status is
+  **REJECTED**.
+
+* The owner can withdraw the PEP by submitting a pull request changing the
+  status to **WITHDRAWN**.
+
+* When progress on the PEP ceases but we aren't ready to reject it, the PEP can
+  be deferred. For example when people are busy for months and but they would
+  like to come back to it later or we want to do it but now isn't the right time
+  because other things need to be done first. In this case the owner or reviewer
+  can submit a pull request changing the status to **DEFERRED**.
+
+  When people are ready to work on it again, the owner should submit a pull
+  request changing the status back to an appropriate state to continue, such as
+  `provisional` or `implementable`.
+
+* Once a PEP has been merged, and a new PEP is started to replace it, or
+  supersede it, a pull request should be opened changing the status to
+  `replaced` and include a link to the new PEP. If the original PEP was
+  implemented, wait to update it to `replaced` until the new PEP is implemented.
+  For in-progress PEPs, they can be changed to `replaced` immediately. The
+  status is now **REPLACED**.
+
+
+## Implementation
+
+When the PEP process is implemented, the contents of the specification section
+will be put into the [CONTRIBUTING.md](/CONTRIBUTING.md) of this repository and
+the Porter repository updated to link to this repository. A PEP template will
+be added to `pep/000-PEP-TEMPLATE.md` as well.
+
+
+## Backwards Compatibility
```

Comment:
I'm confused by these next sections.  Are they the PEP-mandated sections for this particular 001 PEP?

Reply:
I am doing the proposal for a PEP as a PEP. K8s and Helm did the same, essentially testing out the PEP process and using the proposal to have PEPs as the first PEP.

So since part of our PEP requirements is backwards compatability, security, etc you will see those sections filled out as well. The PEP process is outlined in its entirety in the specification section. That is what I will put in the contributing guide in the proposals repository when we agree that the PEP is implementable. The rest of the sections here represent the PEP process as outlined and will just stay in this proposal file.

Reply:
I've added a horizontal rule to help call out where the specification ends.